common/app/model/content.scala

```diff
@@ -833,7 +833,15 @@ final case class Interactive(
     }
   }
 
-  lazy val hasSrcdoc = Jsoup.parseBodyFragment(fields.body).getElementsByTag("iframe").first().hasAttr("srcdoc")
+  lazy val hasSrcdoc = {
+    val iframe = Jsoup.parseBodyFragment(fields.body).getElementsByTag("iframe")
+
+    if (iframe.length > 0) {
```

Comment:
You can achieve this in scala without length checking:

`val iframes = Jsoup.parseBodyFragment(fields.body).getElementsByTag("iframe")
iframes.headOption.exists(_.hasAttr("srcdoc"))
`